frontend/src/components/uam/uam.tsx

```diff
@@ -0,0 +1,23 @@
+import { useEffect, useAppDispatch, useAppSelector } from 'hooks/hooks';
+import { FC } from 'common/types/types';
+import { UsersTable } from './users-table/users-table';
```

Comment:
```suggestion
import { UsersTable } from './components/components';
```

Reply:
Now my import looks like this "import { UsersTable } from './components/users-table/users-table';" because I had to move users-table component into components folder inside uam